Entire-Code

cheatiiit opened the pull request:
@alma-castant @acoconut : Let me know if any changes are required

alma-castant commented on Android/PowerUpApp/hs_err_pid18659.log:
```diff
@@ -0,0 +1,521 @@
+#
```
Do not commit your logs to the repo. Amend the commit removing this file using `git commit --amend`

cheatiiit commented:
Hi @alma-castant  
Removed the log file
Made a single drawable folder
Changed the layout of PowerUpString
Renamed Activities
Removed unused files
I have intentionally kept the dimension separate so that UI widgets are loosely coupled and changes in size of one in future do not effect the other.

alma-castant commented on Android/PowerUpApp/res/layout/avatar_room.xml:
```diff
@@ -0,0 +1,117 @@
+        android:layout_height="@dimen/avatar_room_height"
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentTop="true"
+        android:layout_marginLeft="33dp"
```
Should be a `@dimen` reference

alma-castant commented on Android/PowerUpApp/res/layout/avatar_room.xml:
```diff
@@ -0,0 +1,117 @@
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentTop="true"
+        android:layout_marginLeft="33dp"
+        android:layout_marginTop="57dp"
```
same

alma-castant commented on Android/PowerUpApp/res/layout/avatar_room.xml:
```diff
@@ -0,0 +1,117 @@
+        android:layout_width="@dimen/avatar_room_width"
+        android:layout_height="@dimen/avatar_room_height"
+        android:layout_alignTop="@id/eyes"
+        android:layout_marginLeft="35dp"
```
same

alma-castant commented on Android/PowerUpApp/res/layout/avatar_room.xml:
```diff
@@ -0,0 +1,117 @@
+        android:layout_height="wrap_content"
+        android:layout_alignRight="@id/face"
+        android:layout_below="@id/face"
+        android:layout_marginTop="21dp"
```
same

alma-castant commented on Android/PowerUpApp/res/layout/avatar_room.xml:
```diff
@@ -0,0 +1,117 @@
+        android:layout_height="wrap_content"
+        android:layout_alignLeft="@+id/face"
+        android:layout_below="@id/faceRight"
+        android:layout_marginTop="30dp"
```
same

alma-castant commented on Android/PowerUpApp/res/layout/pgame.xml:
```diff
@@ -4,63 +4,85 @@
-        android:layout_alignRight="@+id/mainListView"
-        android:layout_marginBottom="17dp"
-        android:text="Back to Map" />
+        android:layout_alignBottom="@+id/redoButton"
```
Should be only `@id`

cheatiiit replied:
RedoButton has not been declared before this point. Need this +

alma-castant replied:
Ok

alma-castant commented on Android/PowerUpApp/res/layout/pgame.xml:
```diff
@@ -4,63 +4,85 @@
+        android:layout_width="@dimen/main_list_view_width"
+        android:layout_height="@dimen/main_list_view_height"
+        android:layout_alignParentRight="true"
+        android:layout_below="@+id/askerImageView"
```
Same

cheatiiit replied:
Need the + here

alma-castant replied:
Ok

alma-castant commented on Android/PowerUpApp/src/com/example/powerupapp/Avatar.java:
```diff
@@ -1,40 +1,94 @@
 package com.example.powerupapp;
```
This file can probably be removed now

alma-castant commented on Android/PowerUpApp/src/com/example/powerupapp/Map.java:
```diff
@@ -21,7 +21,7 @@ public void onCreate(Bundle savedInstanceState) {
 		getmDbHandler().open();
```
You probably want to remove this class now

alma-castant commented on Android/PowerUpApp/src/com/example/powerupapp/datamodel/Scenario.java:
```diff
@@ -10,6 +10,7 @@
 	private Integer firstQId;
```
You could probably just turn all elements and getters into `int`

alma-castant commented on Android/PowerUpApp/src/com/example/powerupapp/db/AbstractDbAdapter.java:
```diff
@@ -17,11 +17,11 @@
 	private static final int DATABASE_VERSION = 1;
 	// Database Name
-	private static final String DATABASE_NAME = "DatabaseNamePowerUpApp";
+	private static final String DATABASE_NAME = "DBNamePowerUpAppGSoCProjectchetnaGupta";
```
The database could have a simple name like `PowerUpDB`

cheatiiit commented:
Updated the PR

alma-castant commented on Android/PowerUpApp/res/layout/gamemap.xml:
```diff
@@ -2,57 +2,58 @@
         android:layout_alignParentBottom="true"
         android:layout_alignParentRight="true"
-        android:layout_marginRight="52dp"
+        android:layout_marginRight="@dimen/conitnue_button_map_margin_right"
```
there's a typo in "continue"

cheatiiit commented:
Changed color/white to android:color.white. @alma-castant :)

alma-castant commented:
@rosariorobinson +1
